### PR TITLE
HPCC-20249 Updates to ECL IDE Docs

### PR DESCRIPTION
**Pull request by g-pan**
Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

NOTE: Inclusive of JIRAs   HPCC-20240, HPCC-20243, HPCC-20250, HPCC-20251, HPCC-20258, HPCC-20261, HPCC-20264, and HPCC-20273 
All pertaining to the ECL IDE

@JamesDeFabia please review

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [ ] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:

Integration Build: http://10.240.32.243/view/Docs%207.0/job/DocDevII/

### CODE REVIEW

File: docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml [hunk 1]
```diff
@@ -1301,17 +1323,14 @@
...
 
                 <row>
                   <entry><emphasis role="bold">Open</emphasis></entry>
 
                   <entry>Provides access to the following Open submenus:
-                  <programlisting role="tab"><emphasis role="bold">Open Builder:</emphasis> opens a saved query (.ECL file) into a new ECL Builder window. CTRL-O
-<emphasis role="bold">Open Attribute:</emphasis> opens a dialog to name the attribute to open. 
-                      Enter the fully qualified attribute name, then press the <emphasis
-                        role="bold">OK </emphasis>button.
+                  <programlisting role="tab"><emphasis role="bold">Open Builder:</emphasis> Opens a saved query(.ECL file) in a new ECL Builder window. CTRL-O
```
Comment (JamesDeFabia) [inline docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml]: CTRL+O

File: docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml [hunk 2]
```diff
@@ -1320,18 +1339,25 @@
                 <row>
                   <entry><emphasis role="bold">Save</emphasis></entry>
 
-                  <entry>saves the contents of the window with input focus. An
+                  <entry>Saves the contents of the window with input focus. An
                   ECL Builder window query is saved to an .ECL file. An Editor
                   window saves the ECL file to the Repository. CTRL-S</entry>
```
Comment (JamesDeFabia) [inline docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml]: CTRL+S

File: docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml [hunk 3]
```diff
@@ -1698,7 +1733,17 @@
                   <entry><emphasis role="bold">Find </emphasis></entry>
 
                   <entry>Locates specific text in the currently active window.
-                  CTRL-F</entry>
+                  CTRL-F </entry>
```
Comment (JamesDeFabia) [inline docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml]: CTRL+F

File: docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml [hunk 4]
```diff
@@ -2582,6 +2636,13 @@
 
                   <entry>deletes the current Workunit</entry>
                 </row>
+
+                <row>
+                  <entry><emphasis role="bold"><emphasis role="bold">Close All
+                  Workunits</emphasis></emphasis></entry>
+
+                  <entry>Closes all open the workunits</entry>
```
Comment (JamesDeFabia) [inline docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml]: remove "the"

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Few comments inline.

**Comment by g-pan** [comment]
requested changes made, @JamesDeFabia please review

**Review by JamesDeFabia — APPROVED** [review]
Good to merge

**Pull request merged**